packages/massa-proto-cli/src/ts-gen.ts

```diff
@@ -200,14 +226,13 @@ export interface TransactionDetails {
 
   ${argsSerialization}
 
-  // Send the operation to the blockchain and retrieve its ID
-  const opId = await callSC("${contractAddress}", "${protoFile.funcName}", serializedArgs, coins);
-
-  // Retrieve the events and outPuts from the operation ID
-  // TODO: Retrieve the events and outPuts from the operation ID
-  const outputData = await getOutput(opId);
-
-  return outputData;
+  // Send the operation to the blockchain and retrieve its outputs
+  return await callSC(
+    'Paste your contract address here',
```

Comment:
Why this part is not dynamically filled ?

Reply:
i forgot it